src/setup-fontawesome.js

```diff
@@ -1,5 +1,11 @@
 import { library } from '@fortawesome/fontawesome-svg-core'
-import { faFacebook, faLinkedin, faWhatsapp, faXTwitter } from '@fortawesome/free-brands-svg-icons'
+import {
+    faFacebook,
+    faGithub,
+    faLinkedin,
+    faWhatsapp,
+    faXTwitter,
+} from '@fortawesome/free-brands-svg-icons'
```

Comment:
So I guess there are no copyright/license restrictions that might make using the GitHub icon problematic, right?

Reply:
I hope not, but I don't think so ^^. At least not for this specific use.